oci-seccomp-bpf-hook.go

```diff
@@ -420,3 +425,57 @@ func syscallInProfile(profile *types.Seccomp, syscall string) bool {
 	}
 	return false
 }
+
+func appendArchIfNotAlreadyIncluded(goArch string, profile *types.Seccomp) error {
+	targetArch, err := goArchToSeccompArch(goArch)
+	if err != nil {
+		return errors.Wrap(err, "determine target architecture")
+	}
+	for _, arch := range profile.Architectures {
+		if arch == targetArch {
+			// architecture already part of the profile
+			return nil
+		}
+	}
+	profile.Architectures = append(profile.Architectures, targetArch)
+	return nil
+}
+
+func goArchToSeccompArch(goArch string) (types.Arch, error) {
```

Comment:
Alternatively we could add this helper to seccomp/containers-golang directly.

Reply:
Good idea. Having it over there reduces the risk of regressing when new architectures are added.

Reply:
Alright, opened https://github.com/seccomp/containers-golang/pull/31